ChargerMail/src/Mailbox.java

```diff
@@ -80,12 +67,13 @@ public void removeMessage(Message message) {
 	 * @return true if emptied, false if not.
 	 */
 	public boolean emptyTrash() {
```

Comment:
In general, the Mailbox class should have an emptyMailbox command. Not just for the trash mailbox. We could implement an empty function for inbox and sent box that sends their respective messages to the trash box, and then if the trash box is emptied, those messages are forever lost.